exercises/1-the-manual-menace/README.md

```diff
@@ -436,7 +436,21 @@ where
     * `<GIT_URL>` is the full clone path of the repo where this project is stored (including the https && .git)
     * `<YOUR_NAME>` is the prefix for your `-ci-cd` project.
 
-8. Create a new object `ci-cd-builds` in the Ansible `inventory/host_vars/ci-cd-tooling.yml` to drive the s2i build configuration.
+8. Create `params/jenkins-s2i-slaves` and add the following content; replacing variables as appropriate.
+
```

Comment:
I think this should not be called `s2i-slaves` as it's not s2i, it's a jenkins slave. The param file should just be called `jenkins-slave-npm`